configs/components/rubygem-rubyntlm.rb

```diff
@@ -1,6 +1,16 @@
 component 'rubygem-rubyntlm' do |pkg, settings, platform|
-  pkg.version '0.6.3'
-  pkg.md5sum 'e1f7477acf8a7d3effb2a3fb931aa84c'
+  pkg.environment "GEM_HOME", settings[:gem_home]
+  pkg.environment "GEM_PATH", settings[:gem_home]
+  pkg.url("https://github.com/larskanis/rubyntlm.git")
```

Comment:
We definitely shouldn't do, IMO. Pulling in randos branches that could change out from underneath us. If there's not a released gem for this and we have to build from source I think we should fork this repo and build off of a SHA pointing to our own repo.

Reply:
For sure, this was just to get a build to test. We have options here...